carbonmark-data/components/charts/TokenPricesChart/index.tsx

```diff
@@ -0,0 +1,97 @@
+import { formatTonnes } from "@klimadao/lib/utils/lightIndex";
+import { t } from "@lingui/macro";
+import { ArrowDropDown, ArrowDropUp, InfoOutlined } from "@mui/icons-material";
+import { formatPrice } from "lib/charts/helpers";
+import { queryPrices, queryTokenInfo } from "lib/charts/queries";
+import { PricesItem, Token } from "lib/charts/types";
+import { TOKENS } from "lib/constants";
+import { currentLocale } from "lib/i18n";
+import {
+  getTokenFullName,
+  getTokenIcon,
+  getTokenSelectiveFeeDescription,
+} from "lib/tokens";
+import { CoinTiles, CoinTilesData } from "../helpers/CoinTiles";
+import styles from "./styles.module.scss";
+
+/** Async server component
+ */
+export default async function TokenPricesChart() {
+  const locale = currentLocale();
+  const prices7daysAgo: PricesItem = (
+    await queryPrices({ sort_by: "date", sort_order: "desc", page_size: 8 })
+  ).items[6];
+  const coinTilesData: CoinTilesData = [];
+  for (let i = 0; i < TOKENS.length; i++) {
+    const token: Token = TOKENS[i];
+    const tokenInfo = await queryTokenInfo(token);
+
+    if (tokenInfo) {
+      // Token supply
+      const formattedCurrentSupply = formatTonnes({
+        amount: tokenInfo.current_supply.toString(),
+        locale: locale,
+      });
+      // Price
+      // TODO: Should we localize price?
+      //const price = tokenInfo.price.toFixed(2);
+      const price = formatPrice(locale)(tokenInfo.price);
```

Comment:
This formatting doesn't match the figma designs; it should be "$0.68" instead of "0.68 $"

Reply:
See #1504